ush/hpssarch_gen.sh

```diff
@@ -560,11 +560,12 @@ if [[ ${type} == "enkfgdas" || ${type} == "enkfgfs" ]]; then
 
       MEMDIR="mem${mem}" YMD=${PDY} HH=${cyc} generate_com \
         COM_ATMOS_ANALYSIS_MEM:COM_ATMOS_ANALYSIS_TMPL \
-        COM_ATMOS_RESTART_MEM:COM_ATMOS_RESTART_TMPL
+        COM_ATMOS_RESTART_MEM:COM_ATMOS_RESTART_TMPL \
+        COM_ATMOS_HISTORY:COM_ATMOS_HISTORY_TMPL
```

Comment:
```suggestion
        COM_ATMOS_HISTORY_MEM:COM_ATMOS_HISTORY_TMPL
```

Reply:
Works in testing. Accepting commit suggestion.